docs/src/pages/components/Tables/WebTables.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Tables
+platform: web
+githubPath: bpk-component-table
+---
+
+import {
+  BpkTable,
+  BpkTableHead,
+  BpkTableBody,
+  BpkTableRow,
+  BpkTableHeadCell,
+  BpkTableCell,
+} from 'bpk-component-table';
+
+import Readme from 'components/Readme';
+import PresentationBlock from 'components/PresentationBlock';
+import tablesReadme from 'bpk-component-table/README.md';
+import datatableReadme from 'bpk-component-datatable/README.md';
+
+import { AutowidthExample } from 'bpk-component-datatable/examples';
+
+## Table of contents
+
+## Standard table
+
+A simple table component for displaying data in a tabular form.
+
+<PresentationBlock>
+  <BpkTable>
+    <BpkTableHead>
+      <BpkTableRow>
+        <BpkTableHeadCell>Heading 1</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 2</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 3</BpkTableHeadCell>
+        <BpkTableHeadCell>Heading 4</BpkTableHeadCell>
+      </BpkTableRow>
+    </BpkTableHead>
+    <BpkTableBody>
+      <BpkTableRow>
+        <BpkTableCell>Row 1, Data 1</BpkTableCell>
+        <BpkTableCell>Row 1, Data 2</BpkTableCell>
+        <BpkTableCell>Row 1, Data 3</BpkTableCell>
+        <BpkTableCell>Row 1, Data 4</BpkTableCell>
+      </BpkTableRow>
+      <BpkTableRow>
+        <BpkTableCell>Row 2, Data 1</BpkTableCell>
+        <BpkTableCell>Row 2, Data 2</BpkTableCell>
+        <BpkTableCell>Row 2, Data 3</BpkTableCell>
+        <BpkTableCell>Row 2, Data 4</BpkTableCell>
+      </BpkTableRow>
+    </BpkTableBody>
+  </BpkTable>,
```

Comment:
I guess this storybook example will be migrated in the future?
Maybe worth adding a TODO comment to update it when it's been migrated?

Reply:
Yes this component has not been updated to the new format yet - I guess we will auotmatically updating the examples when we do the change in the web component so might be yagni?